src/colors.ts

```diff
@@ -0,0 +1,75 @@
+import materialColors from 'material-colors-ts'
+import { ArrayElement } from './util'
+
+export const colors = [
+  'pink',
+  'purple',
+  'deepPurple',
+  'indigo',
+  'blue',
+  'lightBlue',
+  'cyan',
+  'teal',
+  'green',
+  'lightGreen',
+  'lime',
+  'yellow',
+  'amber',
+  'orange',
+  'deepOrange',
+  'red',
+] as const
+
+export type Color = ArrayElement<typeof colors>
+
+export type Brightness = 'light' | 'medium' | 'dark'
+
+export function cssColorFromColor(
+  color: Color | 'lightblue' | 'lightgreen' | 'gold',
+  brightness: Brightness,
+): string {
+  switch (color) {
+    case 'lightblue': {
+      return cssColorFromColor('lightBlue', brightness)
+    }
+    case 'lightgreen': {
+      return cssColorFromColor('lightGreen', brightness)
+    }
+    case 'gold': {
+      return cssColorFromColor('orange', brightness)
+    }
+    case 'pink':
+    case 'purple':
+    case 'deepPurple':
+    case 'indigo':
+    case 'blue':
+    case 'lightBlue':
+    case 'cyan':
+    case 'teal':
+    case 'green':
+    case 'lightGreen':
+    case 'lime':
+    case 'yellow':
+    case 'amber':
+    case 'orange':
+    case 'deepOrange':
+    case 'red': {
+      const variant = variantForBrightness(brightness)
+      return materialColors[color][variant]
+    }
+  }
+}
+
+function variantForBrightness(brightness: Brightness): 100 | 200 | 300 {
```

Comment:
Nice trick.

Reply:
eventually this will also take a `theme: 'light' | 'dark'` parameter and return 700, 800, or 900 instead